Source/ASDisplayNode+Beta.h

```diff
@@ -21,7 +21,7 @@
 #import <AsyncDisplayKit/ASEventLog.h>
 
 #if YOGA
-  #import YOGA_HEADER_PATH
+  @import yoga;
```

Comment:
I would like to avoid this change if possible. Does your error still occur if this is not changed? Can you screenshot the error / warning that you get without this change, but with the HEADER_PATH change?

The reason we'd like to avoid this is that some build systems do not support @import. It would be the only usage in the entire framework, and I believe it should be possible to make this work reliably without using @import.